osl/report/raw_report.py

```diff
@@ -350,8 +350,21 @@ def plot_channel_time_series(raw, savebase=None):
 
 def plot_sensors(raw, savebase=None):
     """Plots sensors with bad channels highlighted."""
-
-    fig = raw.plot_sensors(show=False)
+    # plot channel types seperately for neuromag306 (3 coils in same location)
+    if 3012 in np.unique([i['coil_type'] for i in raw.info['chs']]):
+        with open('utils/neuromag306_info.yml', 'r') as f:
+            channels = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
fyi, there's yaml.safe_load if you didn't want to specify the Loader

Reply:
Fixed in https://github.com/OHBA-analysis/oslpy/pull/55/commits/e9140485440b2828de8fdc0846c6dbfd6c4c389e